static/js/portico/signup.js

```diff
@@ -136,14 +136,26 @@ $(() => {
         }
     });
 
-    const show_subdomain_section = function (bool) {
-        const action = bool ? "hide" : "show";
-        $("#subdomain_section")[action]();
+    const subdomain_required = function (bool) {
```

Comment:
I am a fan of function names that tell you what they do (i.e. contain some verb), maybe `update_subdomain_input_requirement`? Besides I think we can just as well use a normal function declaration here.

Reply:
How about `toggle_subdomain_required`?

And I was wondering, why do we declare functions as expressions anyway?

Reply:
If we remove the bool parameter like I suggested in another comment, `update` fits better than `toggle`. I think we should keep `input` in there because after all it's about the input being required.

Me too, that's why I suggested using a regular function declaration.

Reply:
Updated.